lib/openxml/docx/elements/word_processing_shapes_shape_properties.rb

```diff
@@ -0,0 +1,23 @@
+module OpenXml
+  module Docx
+    module Elements
+      class WordProcessingShapesShapeProperties < Container
+        tag :spPr
+        namespace :wps
+
+        attribute :bwMode, expects: :valid_colors
+
+      private
+
+        def valid_colors(value)
+          raise ArgumentError unless ok_values.include? value
+        end
+
+        def ok_values
+          %i(auto black blackGray blackWhite clr gray grayWhite hidden invGray ltGray white)
+        end
```

Comment:
Then, too, we can write a standard helpful message instead of just raising `ArgumentError` with no additional clues.